lib/bolt/project_migrate/base.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'bolt/error'
+
+module Bolt
+  class ProjectMigrate
+    class Base
+      def initialize(outputter)
+        @outputter = outputter
+      end
+
+      protected def print_step(step)
```

Comment:
This seems like it belongs in the human outputter class, especially since we're already passing the outputter around. That's where we handle things like formatting and printing, and would ensure we don't have to redefine `indent` here. I think indent can use the `Bolt::Outputter#indent` method, and wrap can go in that `Bolt::Outputter` as well.